dbfit-java/build.gradle

```diff
@@ -330,7 +330,8 @@ project('oracle') {
 
     dependencies {
         implementation project(':dbfit-java:core')
-		implementation 'dummy:ojdbc8:18.3.0.0.0@jar'		
+        implementation 'dummy:ojdbc6:11.2.0.3.0@jar'
+		// implementation 'dummy:ojdbc8:18.3.0.0.0@jar'		
```

Comment:
We don't need to keep that comment. Line formatting is not quite in sync with style used elsewhere (we use spaces for indentation, not tabs)